src/backend/distributed/commands/database.c

```diff
@@ -192,6 +199,30 @@ PreprocessGrantOnDatabaseStmt(Node *node, const char *queryString,
 }
 
 
+/*
+ * FilterDistributedDatabases filters the database list and returns the distributed ones,
+ * as a list.
+ */
+static List *
+FilterDistributedDatabases(List *databases)
+{
+	List *distributedDatabases = NIL;
+	String *databaseName = NULL;
+	foreach_ptr(databaseName, databases)
+	{
+		bool missingOk = true;
```

Comment:
Please define missingOk variable outside for loop. No need to define the variable in each iteration

Reply:
But we mostly try to keep the variables closer to the places where they're used, and I wouldn't expect a performance impact in doing this. If you don't have a strong objection, I'd like to keep it as is.

Reply:
It's not a strong objection.